src/Symfony/Component/Config/FileLocator.php

```diff
@@ -42,11 +42,19 @@ public function locate($name, $currentPath = null, $first = true)
         }
 
         if ($this->isAbsolutePath($name)) {
-            if (!file_exists($name)) {
+            if (file_exists($name)) {
+                return $name;
+            }
+
+            if (array() === $files = glob($name, defined('GLOB_BRACE') ? GLOB_BRACE : 0)) {
                 throw new FileLocatorFileNotFoundException(sprintf('The file "%s" does not exist.', $name));
             }
 
-            return $name;
+            if (true === $first) {
+                return array_shift($files);
```

Comment:
`array_shift` is used more commonly across the rest of the code base when doing a return. I don't think it makes much of a difference either way